res/values/strings.xml

```diff
@@ -183,133 +183,183 @@ there are any).</string>
+    <string name="np_desc_it">Roma, Milano, Torino, Venezia, Palermo, Trento</string>
+    <string name="np_desc_it_networks" translatable="false">ATM, GTT, AMAT, ACTV</string>
 
     <string name="np_region_liechtenstein">Liechtenstein</string>
     <string name="np_desc_vmobil">Liechtenstein, Vorarlberg, Bregenz</string>
 
-    <string name="np_region_switzerland">Switzerland</string>
-    <string name="np_desc_vbl">Luzern</string>
-    <string name="np_desc_zvv">Zurich</string>
-
-    <string name="np_region_belgium">Belgium</string>
     <string name="np_region_luxembourg">Luxembourg</string>
+    <string name="np_name_lu" translatable="false">Verkéiersverbond</string>
+    <string name="np_desc_lu">Luxembourg</string>
+    <string name="np_desc_lu_networks" translatable="false">AVL, SNCFL, FLEAA, RGTR, TICE</string>
+
     <string name="np_region_netherlands">Netherlands</string>
-    <string name="np_region_denmark">Denmark</string>
-    <string name="np_region_sweden">Sweden</string>
+    <string name="np_desc_ns">Netherlands, Amsterdam</string>
+
     <string name="np_region_norway">Norway</string>
-    <string name="np_region_gb">Great Britain</string>
-    <string name="np_region_ireland">Ireland</string>
-    <string name="np_region_italy">Italy</string>
-    <string name="np_region_poland">Poland</string>
-    <string name="np_region_uae">United Arab Emirates</string>
-    <string name="np_region_usa">United States of America</string>
-    <string name="np_region_australia">Australia</string>
-    <string name="np_region_israel">Israel</string>
-    <string name="np_region_france">France</string>
-    <string name="np_region_finland">Finland</string>
+    <string name="np_desc_nri">Norway, Oslo, Bergen</string>
+
     <string name="np_region_nz">New Zealand</string>
+    <string name="np_name_nz">New Zealand</string>
+    <string name="np_desc_nz">New-Zealand, Welligton, Auckland</string>
+    <string name="np_desc_nz_networks" translatable="false">Go Welligton, Tranz Metro, Auckland Transport</string>
+
+    <string name="np_region_poland">Poland</string>
+    <string name="np_name_pl" translatable="false">PKP</string>
+    <string name="np_desc_pl">Poland, railways in Poland</string>
+
     <string name="np_region_spain">Spain</string>
-    <string name="np_region_br">Brazil</string>
-    <string name="np_region_canada">Canada</string>
+    <string name="np_name_spain">Spain</string>
+    <string name="np_desc_spain">Madrid buses, Barcelona, Basque</string>
+    <string name="np_desc_spain_networks" translatable="false">EMT, TMB, Moveuskadi</string>
 
-    <string name="np_desc_ns">Netherlands, Amsterdam</string>
-    <string name="np_desc_dsb">Denmark, Copenhagen</string>
+    <string name="np_region_sweden">Sweden</string>
     <string name="np_desc_se">Sweden, Stockholm</string>
+    <string name="np_name_stockholm">Stockholm</string>
     <string name="np_desc_stockholm">Stockholm</string>
```

Comment:
I think we can delete the Stockholm strings and the provider. Doesn't look like it is coming back.